.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "go.lintTool": "golangci-lint"
+}
```

```suggestion
}

```
本質じゃないんですが最後改行入れてくれると:pray: